.github/workflows/screenshots.yml

```diff
@@ -0,0 +1,79 @@
+name: Screenshots
+
+on:
+  push:
```

Comment:
This allows us to collect a new baseline (or 'Before' screenshot) each time a new change is merged to `main`.

Reply:
Obviously we'll only know that works correctly for sure once this PR is merged!

Reply:
So if the branch is `main` this runs up to and including the "Upload screenshot" step, but doesn't add the comments?